.github/pull-request-template.md

```diff
@@ -0,0 +1,15 @@
+## Summary
+<!-- A brief summary of the PR -->
+
+## Checklist
+<!-- [ ] = no, [x] = yes -->
+
+ - [ ] I have searched for similar/duplicate PR's, and there are none.
+ - [ ] This is a **documentation update** (main code base unchanged).
+ - [ ] This is a **source code update** (main code base changed).
+   - [ ] I have created unit tests for the modified code base.
+   - [ ] All tests pass (if not please elaborate).
+   - [ ] This is a breaking change (requires major version update).
+
+## Related issues
```

Comment:
I just think that closes is nicer. Just personal preference but I should be able to fix this myself.

```suggestion
## Closes
```